docs/about/compatibility/openmp.md

```diff
@@ -472,3 +472,93 @@ A no-loop kernel is not generated if the OpenMP teams construct uses a `num_team
 #### Cross-team optimized reduction kernel generation
 
 If the OpenMP construct has a reduction clause, the compiler attempts to generate optimized code by utilizing efficient cross-team communication. New APIs for cross-team reduction are implemented in the device runtime and are automatically generated by clang.
+
+### Shared memory support
+
+Unlike other discrete GPU systems that have distinct CPU (system) and GPU memory, MI300A architecture features a single physical storage per socket which is the main memory for both CPU and GPU.
+To support the optimal use of shared memory on MI300A, the ROCm OpenMP compiler and runtime are optimized to use “zero-copy” also known as “no-copy” behavior as the default execution model where the CPU and GPU threads use the same addresses to access the same physical storage. The zero or no-copy behavior also ensures that the OpenMP map constructs do not result in device memory allocation and copies between host and device memory, as there is no physical device storage.
+
+For demonstration purposes, we are using [vec_add.cpp](#sample-program) in the examples below.
+
+To enable zero-copy behavior by default in the programs running on MI300A, follow these options:
+
+* When using `unified_shared_memory` pragma in the program:
+
+You can enable `unified_shared_memory` in your program using two options. One is to use `#pragma omp requires unified_shared_memory` construct in your source files and the other option is to use the compiler option `-fopenmp-force-usm` that enables `unified_shared_memory` without having to write it in the program. Building using each option is explained below but irrespective of the option chosen to enable `unified_shared_memory`, you must build and run the program in an Xnack-enabled environment.
```

Comment:
what is the correct xnack spelling? I thought it is all caps.

Reply:
We have used Xnack previously in the document hence I stuck to the same spelling. Perhaps Carlo can confirm.

Reply:
There are two "xnack" here.
- Compiler target feature (xnack+, xnack-, xnack-any). This is lower case
- Runtime state: XNACK-Enabled; XNACK-Disabled. This is upper case.

Reply:
@carlobertolli Thank you for clarifying. 

@SwRaw Please add this to the wordlist and update the internal page on Confluence with the definitions.

Reply:
updated opemp.md.